bin/hab-verify

```diff
@@ -52,12 +54,7 @@ function in_studio() {
   origin="${HAB_ORIGIN:-chef}"
   export HAB_ORIGIN="$origin"
 
-  # We need to use $1 since we are calling the function as:
-  #
-  # in_studio TASK
-  #
-  # and it because its first parameter, not the global parameter of this script
-  if [[ -z "$1" ]]; then
+  if [[ -z "$*" ]]; then
```

Comment:
If the question where trying to answer here is, "are there any non-empty args?", then we need to allow `$*` to be expanded and split. That means not quoting it and using single brackets for the test. See what happens when we use this construct with two empty positional parameters:
```bash
jugokai:ci-studio-common jbauman$ set -- '' ''
jugokai:ci-studio-common jbauman$ [[ -z "$*" ]]; echo $?
1
jugokai:ci-studio-common jbauman$ [[ -z $* ]]; echo $?
1
jugokai:ci-studio-common jbauman$ [ -z $* ]; echo $?
0
```
I don't think this would've caused any problems in your script, but it's a good opportunity to point out a subtle and non-obvious behavior.